mountaineer/__init__.py

```diff
@@ -1,16 +1,27 @@
-from mountaineer.actions import passthrough, sideeffect  # noqa: F401
-from mountaineer.app import AppController  # noqa: F401
-from mountaineer.config import ConfigBase  # noqa: F401
-from mountaineer.controller import ControllerBase  # noqa: F401
-from mountaineer.dependencies import CoreDependencies  # noqa: F401
-from mountaineer.exceptions import APIException  # noqa: F401
-from mountaineer.js_compiler.postcss import PostCSSBundler  # noqa: F401
-from mountaineer.paths import ManagedViewPath  # noqa: F401
+from mountaineer.actions import passthrough as passthrough
```

Comment:
Re-declaring dependencies via the `from ... import ... as ...` defines them as "exported" for the purposes of pyright. This silences type warnings that were being thrown while trying to import dependencies from the core mountaineer project, like:

```python
from mountaineer import RenderBase
```